fare/theme/assets/scss/fare/theme.scss

```diff
@@ -53,10 +69,39 @@ a{
   margin-right:auto;
 }
 
-.frontpage-icon-header{
-  padding-bottom: 0px
-}
-
 #publishform, #downloadform, #deleteform{
   display: inline;
 }
+
+footer {
+  margin-top: 50%;
+  background-color: lightgrey;
+}
+
+.list-inline{
```

Comment:
All this rules look OK, however are they related to the footer or not? If yes, it could be nicer to use hierarchy, check this out for example: https://www.w3schools.com/css/css_combinators.asp

Reply:
No, they are for the css around the site, only .footer is related to footer